.github/workflows/fast-track.yml

```diff
@@ -0,0 +1,19 @@
+name: Request PR Fast-track
+on:
+  pull_request_target:
+    types: [labeled]
+
+jobs:
+  fastTrack:
+    if: github.repository == 'nodejs/node'
+    runs-on: ubuntu-latest
+    steps:
+      - name: Request Fast-Track
+        if: github.event.label.name == 'fast-track'
+        env:
+            COMMENTS_URL: ${{ github.event.issue.comments_url || github.event.pull_request.comments_url }}
+        run: |
+          curl -X POST $COMMENTS_URL \
```

Comment:
I wonder if the GitHub CLI is available in actions. I think we could use it to send the message